pkgs/by-name/lu/lunarvim/package.nix

```
@@ -0,0 +1,97 @@
+{ fd
+, fetchFromGitHub
+, fzf
+, git
+, lib
+, makeWrapper
+, neovim
+, nodePackages
+, python311
+, ripgrep
+, stdenv
+, tree-sitter
+}:
+let
+  lunar-python = python311.withPackages ( p: with p; [ pynvim ] );
+  runtimeDeps = [
+    fd
+    fzf
+    git
+    neovim
+    ripgrep
+    lunar-python
+    tree-sitter
+  ] ++ (with nodePackages; [
+    neovim
+  ]);
+in
+stdenv.mkDerivation {
+  pname = "lunarvim";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "LunarVim";
+    repo = "LunarVim";
+    rev = "1.3.0";
```

```suggestion
    rev = "refs/tags/${finalAttrs.version}";
```
Could you please use a variable?
We don't want to rewrite `rev` every time we update.